LLVM.Imports.Core.pas

```diff
@@ -2132,6 +2497,163 @@ procedure LLVMStopMultithreaded; cdecl; external CLLVMLibrary;
     @see llvm::llvm_is_multithreaded *)
 function LLVMIsMultithreaded: LongBool; cdecl; external CLLVMLibrary;
 
+// Added by Max 13/09/2019 17:22:45
+(** Return the function this instruction belongs to.
+ *
+ * Note: it is undefined behavior to call this on an instruction not
+ * currently inserted into a function.  see : const Function *llvm::getFunction()
+ *)
+function LLVMGetInstructionFunc(Inst: TLLVMValueRef):TLLVMValueRef;
+function LLVMGetUsesVal(val : TLLVMValueRef): TArray<TLLVMValueRef> ;
+function LLVMGetNextNonDebugInstruction(I: TLLVMValueRef): TLLVMValueRef;
+function LLVMGetPrevNonDebugInstruction(I: TLLVMValueRef): TLLVMValueRef;
+function LLVMDumpValueToStr(v:TLLVMValueRef) : AnsiString;
+function LLVMGetgetFirstInsertionPt(block: TLLVMBasicBlockRef): TLLVMValueRef;
+function LLVMGetFunctionInstructions(F: TLLVMValueRef): TArray<TLLVMValueRef>;
+procedure LLVMPositionBuilderAfter(Builder: TLLVMBuilderRef; Instr: TLLVMValueRef);
+
 implementation
+    uses System.SysUtils;
+
+function LLVMGetInstructionFunc(Inst: TLLVMValueRef):TLLVMValueRef;
+begin
+    Result.Value := nil;
+
+    var bb : TLLVMBasicBlockRef := LLVMGetInstructionParent(Inst);
```

Comment:
no inline variables. Otherwhise this does not compile on Delphi Berlin or older

Reply:
fix ok ;)